chartfx-math/src/main/java/de/gsi/math/ArrayMath2GenBase.java

```diff
@@ -0,0 +1,63 @@
+package de.gsi.math;
+
+//// === BEGIN IMPORTS ===
+
+import de.gsi.dataset.utils.AssertUtils;
+
+/**
+ * Utility class containing static functions for working with arrays of
+ * different types.
+ *
+ * @author Florian Enner
+ * @since 07 Jun 2020
+ */
+//// === END IMPORTS ===
+class ArrayMath2GenBase {
```

Comment:
Codacy found an issue: [The utility class name 'ArrayMath2GenBase' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper|Constants)'](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5780327)